pkg/util/logdir/logdir.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package logdir
+
+import (
+	"flag"
+)
+
+// GetLogDir returns the path to the directory where Antrea logs are written.
+// It will look for the value of the log_dir command-line flag, or default to a
+// well-known OS-dependent location.
+func GetLogDir() string {
+	logDirFlag := flag.CommandLine.Lookup("log_dir")
+	var logDir string
+	if logDirFlag == nil {
+		logDir = antreaWellKnownLogDir
+	} else if len(logDirFlag.Value.String()) == 0 {
+		logDir = logDirFlag.DefValue
```

Comment:
Out of curiosity, what's the default value?

Reply:
That's a good question. I copied the code from support bundle and it seems wrong now. `DefValue` will be an empty string, but it should be `os.TempDir()`. I will wait for the tests to pass then fix before I merge (this code path is not actually used by any test).